react/Modal/ModalCross.jsx

```diff
@@ -1,27 +1,22 @@
 import React from 'react'
 import styles from './styles.styl'
 import cx from 'classnames'
-import { Button } from '../Button'
 import Icon from '../Icon'
 import palette from '../palette'
 
 const ModalCross = ({ onClick, color, className }) => (
-  <Button
-    theme="close"
+  <button
     className={cx(styles['c-modal-close'], className)}
     onClick={onClick}
-    extension="narrow"
     type="button"
-    label="Close"
-    iconOnly
```

Comment:
We lose accessibility here? Screen reader will not be able to tell the close button, right?

Reply:
Ah right, forgot to add that back in.